packages/contracts/tasks/helpers/benchmarks.ts

```diff
@@ -35,25 +36,57 @@ export async function publishBatch(
 
   const messageBatch = Array.from({ length: batchSize }, () => message.asContractParam());
   const publicKeyBatch = Array.from({ length: batchSize }, () => keypair.publicKey.asContractParam());
+  const deployer = await deployment.getDeployer();
 
   let optimalBatchSize = batchSize;
+  let multiplier = 1;
```

Comment:
Just a small question: why multiplier starts at 1 and not at `DEFAULT_GAS_MULTIPLIER` (2) from here https://github.com/privacy-scaling-explorations/maci/blob/0f18eba8739bd3a3ddf954715b7e67c9433dfce4/packages/contracts/hardhat.config.ts#L26 ?

Reply:
Because it's a different logic: you don't need to replace first transaction and it uses default fees.